matrix-meetings-bot/test/model/ExternalData.test.ts

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
I would prefer to put new tests in the `src` folder. I know that we have a mixture of both patterns already, but we will eventually get rid of the `test` folder once we moved all existing tests.

Reply:
moved to src/util